packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -202,6 +202,15 @@ class RunCommand extends RunCommandBase {
         hide: !verboseHelp,
         help: 'No longer require an authentication code to connect to the VM '
               'service (not recommended).')
+      ..addFlag('disable-dds',
```

Comment:
This is a very helpful comment!

 I would suggest hoisting this and the attach comment into flutter_command.dart with something like:

```
void addDdsOptions(bool verboseHelp) {
  argParser.addOption(...)
}
```

And then call that from run and attach.

Reply:
Thank you! Disabling this can cause some very subtle issues so I wanted to make that very clear in the help message.